README.md

```diff
@@ -198,6 +198,19 @@ postcss: function(webpack) {
 }
 ```
 
+#### `addModulesDirectories`
+
+Type: `Array`  
+Default: `[]`
+
+An array of folder names to add to [Node's resolver](https://nodejs.org/api/modules.html#modules_loading_from_node_modules_folders).
```

Comment:
Probably should link to https://github.com/substack/node-resolve instead, since that is the module that is actually used.